examples/6_totp.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+
+###################################
+# WHAT IS IN THIS EXAMPLE?
+#
+# This is a simple TOTP bot that makes use of
+# the team encrypted key-value store, which
+# we interact with using KVStoreClient.
+###################################
+
+import asyncio
+import logging
+import os
+import sys
+from enum import Enum
+
+import pyotp
+
+from pykeybasebot import Bot, EventType
+
+logging.basicConfig(level=logging.DEBUG)
+
+if "win32" in sys.platform:
```

Comment:
ugh. we need to figure out how to pull this down into the framework. not for this PR. just saying it so it's not only in my head. these examples should have as little boilerplate as possible.

Reply:
made this https://github.com/keybase/pykeybasebot/issues/29